src/lokijs.js

```diff
@@ -359,6 +359,14 @@
           }
         }
         return false;
+      },
+
+      $size: function (a, b) {
+        if (!Array.isArray(a)) {
+          return false;
```

Comment:
@techfort I mean, should be something else returned instead of `false`? Or is that ok?